RELEASE-VERSION

```diff
@@ -0,0 +1 @@
+0.11.58+464
```

Comment:
This file is generated by the build process, so we shouldn't have it in the source repository here.

I'm aware there's a problem when this file is lacking though; we need that particular issue resolved.